examples/theme-components/src/containers/Buttons.tsx

```diff
@@ -0,0 +1,136 @@
+import React, { Component, DetailedHTMLProps, TableHTMLAttributes } from 'react'
+import { Button, TextButton, IconButton } from '@repay/cactus-web'
+import { ActionsAdd, NavigationChevronLeft } from '@repay/cactus-icons'
+import { Link, RouteComponentProps } from '@reach/router'
+
+const tableStyle = {
+  textAlign: 'center',
+  width: '30%',
+  marginLeft: '35%',
+  marginRight: '35%',
+  marginTop: '5px',
+} as DetailedHTMLProps<TableHTMLAttributes<HTMLTableElement>, HTMLTableElement>
```

Comment:
```suggestion
} as CSSProperties
```
I think this type can be simplified

Reply:
Ok yeah, I found where we did this here https://github.com/repaygithub/cactus/blob/48305e80cdb405408cb1c74814d51a171451f6a3/examples/standard/src/App.tsx#L12